File: app/src/main/java/edu/com/softserveinc/bawl/dto/IssueHistoryDto.java
package edu.com.softserveinc.bawl.dto;

import java.util.Date;

/**
 * Created by lubko on 12.10.15.
 */
public class IssueHistoryDto {

    private Date date;

    private String changedByUser;

    private int statusID;

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public String getChangedByUser() {
        return changedByUser;
    }

    public void setChangedByUser(String changedByUser) {
        this.changedByUser = changedByUser;
    }

    public int getStatusID() {
        return statusID;
    }

    public void setStatus(int statusID) {
        this.statusID = statusID;
    }



}

File: app/src/main/scala/edu/com/softserveinc/bawl/dto/IssueHistoryDto.scala
package edu.com.softserveinc.bawl.dto

import java.util.Date

import scala.beans.BeanProperty

class IssueHistoryDTO {
  @BeanProperty  var date: Date = null
  @BeanProperty  var changedByUser: String = null
  @BeanProperty  var status: String = ""
}